packages/graphqlgen-templates/yoga/src/generated/graphqlgen.ts

```diff
@@ -141,7 +141,8 @@ export namespace PostResolvers {
 export namespace UserResolvers {
   export const defaultResolvers = {
     id: (parent: User) => parent.id,
-    name: (parent: User) => parent.name,
+    name: (parent: User) =>
+      parent.name === undefined || parent.name === null ? null : parent.name,
```

Comment:
Let's remove the `|| parent.name === null` check

Reply:
The reason why we added the `|| parent.name === null` is because we're considering a field optional in two cases:

- Either it is optional: eg: `field?: string`
- Or the field is of type `field: string | null`

Therefore, we need to handle both cases. What we could to make it less cumbersome, is to use the unsafe `==` type operator. eg: `parent.name == undefined`. This would check for both `null` and `undefined` if I'm not mistaking